man/systemd.network.xml

```diff
@@ -824,6 +830,86 @@
   </refsect1>
 
   <refsect1>
+    <title>[DUID] Section Options</title>
+
+    <para>This section configures the DHCP Unique Idendifier (DUID) value used by DHCP
+    protocol. DHCPv6 client protocol sends the DHCP Unique Identifier and the interface
+    Identity Association Identifier (IAID) to a DHCP server when acquiring a dynamic IPv6
+    address. DHCPv4 client protocol sends IAID and DUID to the DHCP server when acquiring
+    a dynamic IPv4 address if <option>ClientIdentifier=duid</option>. IAID and DUID allows a
+    DHCP server to uniquely identify the machine and the interface requesting a DHCP IP.</para>
+
+    <para>The DUID value specified here overrides the DUID that systemd-networkd generates
+    using the machine-id from the <filename>/etc/machine-id</filename> file, as well as the
+    global DUID that may be specified in <citerefentry><refentrytitle>networkd.conf
+    </refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+
+    <para>The configured DHCP DUID should conform to the specification in 
+    <ulink url="http://tools.ietf.org/html/rfc3315#section-9">RFC 3315</ulink>,
+    <ulink url="http://tools.ietf.org/html/rfc6355">RFC 6355</ulink>.</para>
+
+    <para>The following options are available in <literal>[DUID]</literal> section:</para>
+
+    <variablelist class='network-directives'>
+
+      <varlistentry>
+        <term><varname>Type=</varname></term>
+        <listitem><para>The type of DUID specified in this section. The following values are
+        supported:</para>
+        <para>raw : If <literal>Type=raw</literal>, then <literal>RawData=</literal> specifies
+        the entire DUID. For e.g: <literal>RawData=00:02:00:00:ab:11:f9:2a:c2:77:29:f9:5c:00</literal>
```

Comment:
Is this colon-separated syntax generally accepted, or something made up here? If so, I'd prefer if we'd just take normal hex chars, without colons, the way we generate it otherwise with hexmem() or parse it with unhexmem().